pyaedt/icepak.py

```diff
@@ -3304,26 +3304,49 @@ def assign_stationary_wall(
             or ``"Heat Transfer Coefficient"``.
         name : str, optional
             Name of the boundary condition. The default is ``None``.
-        temperature : str or float, optional
+        temperature : str or float or dict, optional
             Temperature to assign to the wall. This parameter is relevant if
             ``ext_condition="Temperature"``. If a float value is specified, the
-            unit is degrees Celsius. The default is ``"0cel"``.
-        heat_flux : str or float, optional
+            unit is degrees Celsius. A dictionary can be used for transient
+             assignment. The dictionary should contain three keys: ``"Type"``, ``"Function"`` and
```

Comment:
> I've suggested using a bulleted list to explain how to supply the values for each key. (A blank line must be inserted before the first item and after the last item in the list to display properly in the doc.)

The only problem is the coherency across different BC functions. These lines are copy-pasted from another BC function. I don't want the user to think that the way to create these dictionaries is different just because the documentation "looks" different. 
The solution I want to implement (in the near future - as soon as I complete all bc functions support) is to create a function that helps the user in creating these dictionaries programmatically. Once I create that, we can remove all these lines repeated across the various functions and write properly the documentation just once.